### ISSUE TITLE
Namespace auto add should prefix with a slash

### ISSUE THREAD

**Issue by TheColorRed** [HvyIndustries/crane#246]
When Autocomplete adds a namespace, it adds it as so:

```php
$myClass = new Path\To\MyClass;
```

Which is usually invalid, especially if there is a namespace in the file.

```php
namespace My\App;

class MyOtherClass {
	public function __construct(){
		$myClass = new Path\To\MyClass;
	}
}
```

The above code then trys to access:

```php
// File it attempts to load:
My\App\Path\To\MyClass;

// File it should load:
\Path\To\MyClass;
```

So basically it should prefix the item with a `\`.

**Comment by nevadascout** [comment]
Oops! Thanks for letting me know!

**Comment by nevadascout** [comment]
Fixed in v0.3.2
